spec/mailers/referee_mailer_spec.rb

```diff
@@ -4,7 +4,7 @@
   subject(:mailer) { described_class }
 
   describe 'Send request reference email' do
-    let(:application_form) { create(:completed_application_form) }
+    let(:application_form) { create(:completed_application_form, first_name: 'Harry', last_name: 'Potter') }
```

Comment:
I saw this one fail late yesterday when there was an apostrophe in the generated name - is that the same issue?